diff -r 5fdff160305c src/Makefile.linux
--- a/src/Makefile.linux	Wed Jul 29 18:13:11 2009 +0200
+++ b/src/Makefile.linux	Wed Feb 06 17:18:03 2013 -0500
@@ -36,7 +36,7 @@
 
 # Commands
 
-CC = gcc $(DEFS) -I $(INCDIR) -Wall
+CC = gcc $(DEFS) -I $(INCDIR) -Wall $(CFLAGS)
 RM = rm -f
 STRIP = strip
 
diff -r 5fdff160305c src/oslib_linux/Makefile
--- a/src/oslib_linux/Makefile	Wed Jul 29 18:13:11 2009 +0200
+++ b/src/oslib_linux/Makefile	Wed Feb 06 17:18:03 2013 -0500
@@ -1,6 +1,6 @@
 INCDIR = ../
 
-CC = gcc -Wall -I $(INCDIR) -DPLATFORM_LINUX
+CC = gcc -Wall -I $(INCDIR) -DPLATFORM_LINUX $(CFLAGS)
 AR = ar -ru
 RM = rm -f
