diff --git a/src/crashmail/logwrite.c b/src/crashmail/logwrite.c
index 62877a0..8ef578d 100644
--- a/src/crashmail/logwrite.c
+++ b/src/crashmail/logwrite.c
@@ -113,6 +113,8 @@ void LogWrite(ulong level,ulong category,uchar *fmt,...)
    vprintf(fmt,args);
    printf("\n");
 
+   va_end(args);
+
    time(&t);
    tp=localtime(&t);
 
@@ -125,6 +127,8 @@ void LogWrite(ulong level,ulong category,uchar *fmt,...)
       tp->tm_min,
       tp->tm_sec);
 
+   va_start(args, fmt);
+
    osVFPrintf(logfh,fmt,args);
    osFPrintf(logfh,"\n");
